src/main/java/com/automationpractice/pageobjects/MainPage.java

```diff
@@ -41,22 +41,29 @@ public boolean isPopularLinkActive() {
         return driver.findElements(By.cssSelector("#homefeatured > li"));
     }
 
-    public List<WebElement> getPopularImages() {
-        return driver.findElements(By.cssSelector("#homefeatured > li > div > div.left-block > div"));
+    public List<WebElement> getPopularImage() {
```

Comment:
getPopularImage suggest that method returns one element... and the method returns list

Reply:
Yes, you are right, I changed the name of the method.